src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdEventController.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers
+
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.http.ResponseEntity
+import org.springframework.http.ResponseEntity.ok
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse
+
+@RestController
+class CcdEventController {
+    @PostMapping(
+        value = ["/callback/{type}"],
+        consumes = [APPLICATION_JSON_VALUE],
+        produces = [APPLICATION_JSON_VALUE]
+    )
+    fun handleCallback(
+        @PathVariable("type") type: String,
+        @RequestBody callback: CallbackRequest
+    ): ResponseEntity<CallbackResponse> = ok().build()
+}
```

Comment:
hmm. i'm pretty sure if one of those managing directors of technical chief conglomeration.. thing.. will not go by source code being changed to anything else but java

Reply:
probably best discussed offline but why not try it out?
i.e. not a let's do it on all projects but try it out so we have the information on if / how it is better.

I quite like the changes here in simplifying the tests they do look really good to me

Perhaps separate the PRs,
One for the test refactoring
The other for this class and the new test for it?